payments-core/src/main/java/com/stripe/android/cards/DefaultStaticCardAccountRanges.kt

```diff
@@ -102,21 +102,27 @@ class DefaultStaticCardAccountRanges : StaticCardAccountRanges {
             )
         }
 
+        // https://www.discoverglobalnetwork.com/content/dam/discover/en_us/dgn/docs/IPP-VAR-Enabler-Compliance.pdf
         @VisibleForTesting
         internal val UNIONPAY16_ACCOUNTS = setOf(
```

Comment:
This set of accounts does not contain 3568400000000000, which makes me wonder where Stripe keeps these bins?